commands/local/generate/fixture.js

```diff
@@ -8,6 +8,12 @@ module.exports.builder = function builder(yargs) {
     })
     .positional('fixture-name', {
       describe: 'the name of the fixture to generate',
+    })
+    .option('type', {
```

Comment:
> Should we call this extension instead of type?

I used `type` as the codemod generator also has a `type`, and it passes its `type` as `type` when generating the basic fixture. So I thought it would be consistent. But I can change, also feel not strongly about it...

> Also, should we emit an error (and do nothing) when you attempt to generate a --type=hbs for a codemod-name that is of --type=js?

How would you know the codemod's type? It's JavaScript code, so you would have to parse it somehow, right? Seems hard / error prone / not much bang for the buck. Or do I miss something?

Reply:
> Seems hard / error prone / not much bang for the buck. Or do I miss something?

Nope, I don't think you're missing anything, but I think this is probably something that we should tackle before releasing. Maybe exporting something special?

Reply:
> Maybe exporting something special?

Yes, that makes sense. I did that now, but for the assertion of having fixture and transform of the same type, I did things a bit different now, see the other comment below!